### DISCUSSION TITLE
Disable bypass for a single domain

### DISCUSSION THREAD

**Discussion by CyberBotX** [FastForwardTeam/FastForward#680] [Q&A] [answered]
I was looking at the FAQ as well as the example script in the Custom Bypasses on installing the extension, but I cannot seem to see a way to allow FastForward to not bypass for a single domain. Is it possible for this to be done in a custom bypass script?

**Comment by lostdusty** [comment]
Currently, you can't disable bypasses for website X or Y
This is planned for the next big release of FastForward (see https://github.com/FastForwardTeam/FastForward/issues/656)

**Comment by lostdusty** [comment, marked as answer]
With the MV3 version of the extension, you can now add websites to whitelist.
